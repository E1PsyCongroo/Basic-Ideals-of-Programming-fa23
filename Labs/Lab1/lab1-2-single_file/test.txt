~~~~~~~~~~~~~
hello wrold
~~~~~~~~~~~~~
hello vidar
~~~~~~~~~~~~~
hello hduhelp
~~~~~~~~~~~~~
hello weyoung
~~~~~~~~~~~~~
good bye ZZM
~~~~~~~~~~~~~