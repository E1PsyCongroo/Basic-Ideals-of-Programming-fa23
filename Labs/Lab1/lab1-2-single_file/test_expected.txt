hello wrold
hello vidar
hello hduhelp
hello weyoung
